adminconsole.adoc

```diff
@@ -142,16 +142,16 @@ guide.
 
 image::images/administration_console_ui.png[Administration Console]
 
-<1> Go to Previous or Next page
-<2> Current Location/Path
-<3> Search
-<4> Screen Refresh
-<5> Current User and Logout Option
-<6> Help
-<7> Gear Icon
-<8> Status Pane
-<9> Viewing Pane
-<10> Navigation Pane
+    <1> Go to Previous or Next page
```

Comment:
I'd recommend using unicode dingbats "negative circled <style> digits" (aka white number on black circles) U+278A (➊) - U+2793 (➓) here and on the lists below.
See https://en.wikipedia.org/wiki/Dingbat#Compact_table or http://www.fileformat.info/info/unicode/block/dingbats/list.htm

Reply:
Roger that.  I am investigating alternatives at this time as well for the other things.

Reply:
I'm going to punt on updating this to the unicode dingbats at this time.  My first attempt didn't work and will require additional research to make it work with asciidoctor it seems.
I have opened ZCS-4109 to address the formatting and styling such as this request.